shared/actions/profile.js

```diff
@@ -1,10 +1,17 @@
 // @flow
 import * as Constants from '../constants/profile'
-import type {AsyncAction} from '../constants/types/flux'
-import {apiserverPostRpc} from '../constants/types/flow-types'
-import {navigateUp} from '../actions/router'
+import engine from '../engine'
+import type {Dispatch, AsyncAction} from '../constants/types/flux'
+import type {PlatformsExpanded} from '../constants/types/more'
+import type {UpdateUsername, UpdatePlatform, Waiting, WaitingRevokeProof, FinishRevokeProof} from '../constants/profile'
+import {apiserverPostRpc, proveStartProofRpc, revokeRevokeSigsRpc} from '../constants/types/flow-types'
+import {bindActionCreators} from 'redux'
+import {constants as RpcConstants} from '../constants/types/keybase-v1'
+import {navigateUp, routeAppend} from '../actions/router'
 
-export function editProfile (bio: string, fullname: string, location: string) : AsyncAction {
+const InputCancelError = {desc: 'Cancel Add Proof', code: RpcConstants.StatusCode.scinputcanceled}
```

Comment:
This part (beginning with line 12 and ending with line 130) is from the shared commit and should not be reviewed in context of this PR